### PR TITLE
Adding .NET Core project with example on how to generate signature & …

### PR DESCRIPTION
**Pull request by masaintmartin**
…call /api/v3/account

### CODE REVIEW

**Comment by masaintmartin** [comment]
Proposing my implementation for .NET Core

**Comment by chairz** [comment]
Hi masaintmartin, would be great if you can add a README file to show how we setup and run this program. Thanks!

File: net_core/.gitignore [hunk 1]
```diff
@@ -0,0 +1,405 @@
+# globs
+Makefile.in
```
[the remark this answers is not in the document]
Comment (masaintmartin) [reply net_core/.gitignore:2]: Honestly, i just took a template somewhere. Might Visual Studio one's.

[the remark this answers is not in the document]
Comment (2pd) [reply net_core/.gitignore:2]: Please only add files that related to this script.

[the remark this answers is not in the document]
Comment (chairz) [reply net_core/.gitignore:2]: You can try removing the ignore file and see if it breaks anything which i doubt so.

**Comment by 2pd** [comment]
Sorry we didn't get feedback from author, but submit our own example code of dotnet

**Pull request closed without merging**